modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -21,6 +21,7 @@ The following table describes the CSI drivers that are installed with {product-t
 |OpenStack Cinder | ✅ | ✅ | ✅
 |OpenStack Manila | ✅ | ✅ | ✅
 |Red Hat Virtualization (oVirt) | - | - | -
+|vSphere (Tech Preview) | - | - | -
```

Comment:
Should probably use "VMware vSphere" for consistency in naming both the provider and the platform.